beacon_node/lighthouse_network/src/metrics.rs

```diff
@@ -184,3 +187,61 @@ pub fn scrape_discovery_metrics() {
     set_gauge(&DISCOVERY_SENT_BYTES, metrics.bytes_sent as i64);
     set_gauge(&DISCOVERY_RECV_BYTES, metrics.bytes_recv as i64);
 }
+
+/// Agregated `BandwidthSinks` of tcp and quic transports
+/// used in libp2p.
+pub struct AgregatedBandwithSinks {
```

Comment:
```suggestion
pub struct AggregatedBandwidthSinks {
```

Reply:
thanks @jmcph4 for all these suggestions!